### PR TITLE
Added simple command to build samples and updated command line args for multichannel fd

### PR DESCRIPTION
**Pull request by mahinlma**
updated command line args with respect to web cam configuration 
set -nc 0, when no web cams configured.

### CODE REVIEW

**Comment by mahinlma** [comment]
In CMakeList, for this `CMAKE_BUILD_TYPE STREQUAL "" ` condition, CMAKE_BUILD_TYPE setted as Release.
So added simple command to build samples:
cd to sample directory - 
```
mkdir build && cd build
cmake .. 
make
```

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/README.md [hunk 1]
```diff
@@ -144,7 +144,13 @@ cd build
...
 make
 ```
-
+or  
+Use this simple command which takes default build type `Release`:
+```sh
+mkdir build && cd build  
+cmake ..  
+make
+```
```
Comment (IRDonch) [inline demos/README.md]: I don't quite understand the purpose of this addition. These are exactly the same commands as above, except without the `-DCMAKE_BUILD_TYPE=Release` argument.

File: demos/multichannel_demo/fd/README.md [hunk 2]
```diff
@@ -100,12 +100,13 @@ You can use `-i` option to connect all the three web cameras:
 
 Alternatively, you can just set `-nc 3`, which simplifies application usage.
 
-If your cameras are connected to PC with indexes gap (for example, `0,1,3`), use the `-i` parameter.
+If your cameras are connected to PC with indexes gap (for example, `0,1,2`), use the `-i` parameter.
```
Comment (IRDonch) [inline demos/multichannel_demo/fd/README.md]: This is an incorrect change - the sequence 0,1,2 doesn't have a gap in it.

File: demos/multichannel_demo/fd/README.md [hunk 3]
```diff
@@ -100,12 +100,13 @@ You can use `-i` option to connect all the three web cameras:
...
 
 IP сameras support:
 ```
 -i rtsp://camera_address_1/ rtsp://camera_address_2/
 ```
+ note: set `-nc 0`, when web cam not configured/connected.
```
Comment (IRDonch) [inline demos/multichannel_demo/fd/README.md]: But `-nc` already defaults to zero. Why set it explicitly?

File: demos/README.md (hunk 1 above)
Comment (mahinlma) [reply demos/README.md]: for this CMAKE_BUILD_TYPE STREQUAL "" condition, CMAKE_BUILD_TYPE setted as Release In CMakeList. so we don't require to add `-DCMAKE_BUILD_TYPE=Release` explicitly for release Configuration

Comment (IRDonch) [reply demos/README.md]: True, but does that warrant creating a duplicate set of instructions? It's not that difficult to add `-DCMAKE_BUILD_TYPE=Release`, even if it is redundant.

**Pull request closed without merging**